src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -3,57 +3,43 @@
   "persons" : [ {
     "name" : "Alice Pauline",
     "studentId": "A0223456X",
-    "phone" : "94351253",
     "email" : "e0345678@u.nus.edu",
-    "address" : "123, Jurong West Ave 6, #08-111",
     "grade" : "A+",
     "groups" : [ "Group 1" ]
   }, {
     "name" : "Benson Meier",
     "studentId": "A0323456X",
-    "phone" : "98765432",
     "email" : "e1023456@u.nus.edu",
-    "address" : "311, Clementi Ave 2, #02-25",
     "grade" : "A",
     "groups" : [ "Group 2B", "Group 1" ]
```

Comment:
Since the code currently only allows 1 person to 1 group. Should we keep the test case having 2 groups assigned?

Reply:
I think it is still possible to assign a student to >1 groups using add command

Reply:
okay lgtm then